ApiBundle/Controller/ContentController.php

```diff
@@ -180,8 +180,9 @@ public function deleteAction($contentId)
     public function duplicateAction(Request $request, $contentId)
     {
         /** @var ContentInterface $content */
-        $content = $this->findOneContent($contentId, $request->get('language'));
-        $newContent = $this->get('open_orchestra_backoffice.manager.content')->duplicateContent($content);
+        $content = $this->findOneContent($contentId, $request->get('language'), $request->get('version'));
+        $lastVersion = $this->getContentLastVersion($contentId, $request->get('language'));
```

Comment:
this should not be done in the controller!

The controller should only be aware of the manager